### PR TITLE
Update documentation for option auto_set_status_to_assigned (target 1.3.x)

### PR DESCRIPTION
**Pull request by cproensa**
Fixes: 21415
Note: this commit can be cherry-picked cleanly into master

### CODE REVIEW

File: docbook/Admin_Guide/en-US/Page_Descriptions.xml [hunk 1]
```diff
@@ -795,7 +795,7 @@
                             <row>
                                 <entry>Status to set auto-assigned issues to</entry>
                                 <entry>$g_bug_assigned_status</entry>
-                                <entry>status issue is set to when assigned</entry>
+                                <entry>status issue is set to when assigned (when option $g_auto_set_status_to_assigned is enabled)</entry>
```
Comment (vboctor) [inline docbook/Admin_Guide/en-US/Page_Descriptions.xml]: How about "Status to use when an issue is auto-assigned"?

**Comment by vboctor** [comment]
:+1:

**Comment by cproensa** [comment]
changed with @vboctor suggestion, now the description is clearer.

**Comment by vboctor** [comment]
Merged via b0a64aad75e428020af10a99fe84b7aa5decfc1a for master and 3c4c165a0a1f880266db0fa86cb545879d1c0152 for master-1.3.x.

**Pull request closed without merging**